.github/workflows/regression.yaml

```diff
@@ -286,6 +286,26 @@ jobs:
           cd build
           ctest --output-on-failure
 
+  amiga:
+    strategy:
+       matrix:
+         include: [
+           {host: "ppc-amigaos"},
+           {host: "ppc-morphos", cflags: "-noixemul"},
```

Comment:
This way, certain configury-emitted flags aren't appended to CFLAGS.
We should instead set `CC="ppc-morphos-gcc -noixemul"`

Reply:
OK, done.